packages/server/src/fhir/repo.ts

```diff
@@ -360,6 +362,13 @@ export class Repository extends FhirRepository<PoolClient> implements Disposable
     if (this.isSuperAdmin()) {
       return true;
     }
+    console.log(
```

Comment:
Should we use `globalLogger` here? Or was this intended for debugging?

Reply:
Just for debugging, thanks for flagging!